### ISSUE TITLE
Next: accessible names need

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#465]
The burger menu (mobile) doesn't have an accessible name.

The same issue is on all the three points dropdown icons on the cards.
